controllers/users.js

```diff
@@ -410,6 +410,31 @@ const updateSelf = async (req, res) => {
       }
     }
 
+    if (req.body.disabled_roles) {
+      let rolesToDisable;
+      const data = req.body.disabled_roles;
```

Comment:
Please follow camelCasing for naming in backend?

the DB field can have snake case